### DISCUSSION TITLE
Provider Demographic Information

### DISCUSSION THREAD

**Discussion by jturan** [CMSgov/price-transparency-guide#13] [Ideas]
In the [negotiated rate details object](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-rate-details-object), provider names, addresses, and phone numbers would ideally be provided in the 'providers' array along with the NPIs. Including these important data points would prevent users from needing to join to another (potentially outdated or inaccurate) data source to append demographic information for the NPIs. The CMS NPPES database has been well-documented to often contain out of date information, so most plans maintain their own set of demographic information for facilities and professionals and are required to make it available for members to find in-network providers.

For each NPI, including these fields would address the above concerns: Provider Name, Address 1, Address 2, City, State, Zip Code, Phone Number, Specialty.

**Comment by shaselton-usds** [comment]
Definitely makes sense. 

We did discuss the possibility of requiring contextual data (like everything that you have listed) along with the NPI, but felt that the initial requirement asks along with the file size shaping to be potentially quite large that having extra data that could be linked by an NPI would be overly burdensome -- at least for this initial version.

There are existing services the provide provider directories if NPPES is deemed too inaccurate or not timely enough for whatever services that are being built.
